src/transformers/models/auto/modeling_flax_auto.py

```diff
@@ -110,11 +114,18 @@
         (RobertaConfig, FlaxRobertaForMaskedLM),
         (BertConfig, FlaxBertForMaskedLM),
         (BigBirdConfig, FlaxBigBirdForMaskedLM),
-        (BartConfig, FlaxBartForConditionalGeneration),
```

Comment:
@patil-suraj, I don't think we should have Bart in the masked-lm auto class even if this is the case for PyTorch. It should be in pretraining and seq2seq LM IMO.